test/integration/crcsuite/crcsuite.go

```diff
@@ -126,6 +130,29 @@ func CheckClusterOperatorsWithRetry(retryCount int, retryWait string) error {
 	return fmt.Errorf("Some cluster operators are still not running.\n")
 }
 
+func CheckHTTPResponseWithRetry(retryCount int, retryWait string, address string, expectedStatusCode int) error {
+
+	retryDuration, err := time.ParseDuration(retryWait)
+	if err != nil {
+		return err
+	}
+
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
This is going to set InsecureSkipVerify for all users of the DefaultTransport in the current process. Since this might be unexpected, it's probably better to use our own client:
```
    tr := &http.Transport{
        TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
    }
    client := &http.Client{Transport: tr}
    _, err := client.Get("https://golang.org/")
```